dev/com.ibm.ws.security.openidconnect.clients.common/resources/com/ibm/ws/security/openidconnect/clients/common/resources/OidcClientMessages.nlsprops

```diff
@@ -250,4 +250,20 @@ ERROR_GETTING_USERINFO_OR_EXTRACTING_CLAIMS=CWWKS1540E: The {0} OpenID Connect c
 ERROR_GETTING_USERINFO_OR_EXTRACTING_CLAIMS.explanation=The response from the UserInfo endpoint might not be in a format that is expected. The OpenID Connect client might have encountered an error while submitting the UserInfo request.
 ERROR_GETTING_USERINFO_OR_EXTRACTING_CLAIMS.useraction=Check the error message for more information. Verify that the content of the UserInfo response matches the Content-Type HTTP header.
 
+BACKCHANNEL_LOGOUT_REQUEST_FAILED=CWWKS1541E: The back-channel logout request sent to [{0}] encountered an error. {1}
+BACKCHANNEL_LOGOUT_REQUEST_FAILED.explanation=The request does not use the HTTP POST method, the request is missing a logout token or the logout token cannot be validated, or the logout request did not complete successfully.
+BACKCHANNEL_LOGOUT_REQUEST_FAILED.useraction=Verify that the HTTP POST method is used to submit the back-channel logout request. Otherwise, see the user action for the error that is included in this message.
+
+BACKCHANNEL_LOGOUT_REQUEST_MISSING_PARAMETER=CWWKS1542E: The back-channel logout request is not valid because the logout_token parameter is missing or empty.
```

Comment:
Probably this would need a directive to translators, like

`# Do not translate "logout_token"`